napari/utils/shortcuts.py

```diff
@@ -27,6 +27,7 @@
     'napari:activate_labels_pan_zoom_mode': [KeyCode.Digit5],
     'napari:activate_labels_transform_mode': [KeyCode.Digit6],
     'napari:new_label': [KeyCode.KeyM],
+    'napari:set_background_label': [KeyCode.KeyB],
```

Comment:
What about `set_label_to_background`? The current name to me sounds like "change the background label to some other value".

Reply:
Makes sense, I'll make this change later today.